### PR TITLE
Optimize assets

### PR DESCRIPTION
**Pull request by dlackty**
- Remove unused fontawsome fonts in vendor directory (switched to gem bundled version)
- Optimize SVGs (using sago)
	* icon.svg: 1.291 KiB - 64.8% = 0.455 KiB
	* logo.svg: 5.839 KiB - 63.2% = 2.148 KiB
- Optimize other images (using ImageOptim)
	* Saved 190KB out of 665.8KB. 33.5% per file on average.

### CODE REVIEW

File: app/assets/images/icon.svg [hunk 1]
```diff
@@ -1,10 +1 @@
...
-    </g>
-</svg>
\ No newline at end of file
+<svg width="51" height="50" viewBox="0 0 51 50" xmlns="http://www.w3.org/2000/svg"><title>LOGO</title><path d="M40.63 50H1.337C.598 50 0 49.413 0 48.688V1.312C0 .588.6 0 1.338 0h48.324C50.402 0 51 .587 51 1.312v20.592c-.194-.587-2.463-7.31-5.97-10.726-3.67-3.573-10.323-6.496-9.334-4.146.99 2.35.99 8.47.99 8.47l-9.962 3.8-1.495 5.526-12.193 3.215s.87 6.218 4.374 7.305c2.248.697 16.982 2.323 16.982 2.323L31.595 50h9.034z" fill="#A873D1" fill-rule="evenodd"/></svg>
```
Comment (brendastorer) [inline app/assets/images/icon.svg]: Can we take this optimization a step further and add better content for the `<title>`? I think `Hound icon` would work here.

File: app/assets/images/logo.svg [hunk 2]
```diff
@@ -1,19 +1 @@
...
....5-2.64-1.237-3.754-2.213-1.115-.975-2.014-2.2-2.697-3.675-.682-1.476-1.024-3.212-1.024-5.208V11.918zm6.963 25.594c0 1.68.466 2.915 1.4 3.71.932.793 2.036 1.19 3.31 1.19s2.378-.397 3.31-1.19c.933-.795 1.4-2.03 1.4-3.71V11.918c0-1.68-.467-2.915-1.4-3.71-.932-.793-2.036-1.19-3.31-1.19s-2.378.397-3.31 1.19c-.934.795-1.4 2.03-1.4 3.71v25.594zm47.736.544c0 1.588-.295 3.063-.887 4.425-.59 1.362-1.4 2.553-2.423 3.574-1.024 1.02-2.218 1.827-3.584 2.417-1.365.59-2.82.885-4.368.885-1.547 0-3.004-.295-4.37-.884-1.364-.59-2.56-1.395-3.583-2.416-1.023-1.02-1.83-2.212-2.423-3.573-.59-1.36-.887-2.836-.887-4.424V.483h6.962v36.893c0 1.724.41 2.995 1.23 3.81.818.818 1.842 1.227 3.07 1.227 1.23 0 2.253-.41 3.072-1.226.82-.816 1.228-2.087 1.228-3.81V.482h6.963v37.573zM161.076.483h6.69l10.51 29.2h.138V.484h6.962v48.464h-6.553l-10.648-29.133h-.137v29.133h-6.962V.483zm33.264 0h10.307c3.96 0 6.985 1.1 9.078 3.3 2.094 2.202 3.14 5.277 3.14 9.224v22.667c0 4.538-1.103 7.884-3.31 10.04-2.208 2.155-5.382 3.233-9.523 3.233h-9.693V.483zm6.962 41.93h3.208c1.957 0 3.345-.488 4.164-1.464.82-.977 1.23-2.508 1.23-4.595V13.007c0-1.905-.388-3.38-1.162-4.424-.773-1.044-2.184-1.565-4.232-1.565h-3.208v35.395z"/></g></svg>
```
Comment (brendastorer) [inline app/assets/images/logo.svg]: Here we could change the `<title>` to `Hound logo`.

**Comment by brendastorer** [comment]
Thanks @dlackty! These are all terrific optimizations.

**Comment by dlackty** [comment]
@brendastorer Updated! Please have a look. Thank you! :+1:

File: app/assets/images/icon.svg [hunk 3]
```diff
@@ -1,10 +1 @@
...
-    </g>
-</svg>
\ No newline at end of file
+<svg width="51" height="50" viewBox="0 0 51 50" xmlns="http://www.w3.org/2000/svg"><title>Houd icon</title><path d="M40.63 50H1.337C.598 50 0 49.413 0 48.688V1.312C0 .588.6 0 1.338 0h48.324C50.402 0 51 .587 51 1.312v20.592c-.194-.587-2.463-7.31-5.97-10.726-3.67-3.573-10.323-6.496-9.334-4.146.99 2.35.99 8.47.99 8.47l-9.962 3.8-1.495 5.526-12.193 3.215s.87 6.218 4.374 7.305c2.248.697 16.982 2.323 16.982 2.323L31.595 50h9.034z" fill="#A873D1" fill-rule="evenodd"/></svg>
```
Comment (brendastorer) [inline app/assets/images/icon.svg]: Small typo. The "n" is missing in "Hound".

Comment (dlackty) [inline app/assets/images/icon.svg]: Updated again.

**Comment by brendastorer** [comment]
@dlackty Looks great. Thanks again!

**Comment by brendastorer** [comment]
Merged as commit 21a581bfd98702b818ac249e5db550aa6f36dc9e. Thank you @dlackty!

**Pull request closed without merging**